static/app/components/nav/config.tsx

```diff
@@ -209,6 +212,11 @@ export function createNavConfig({organization}: {organization: Organization}): N
       {label: t('Alerts'), to: `/${prefix}/alerts/rules/`, icon: <IconSiren />},
     ],
     footer: [
+      {
+        label: t('Help'),
+        icon: <IconQuestion />,
+        overlay: lazy(() => import('./overlay/help')),
```

Comment:
Why have lazy loading for this?

Reply:
Switched the configuration to a POJO in [e69afcc](https://github.com/getsentry/sentry/pull/79720/commits/e69afcc68bdf675e2f1f6044e054ad9573d6cf6b)